docker-compose.yml

```diff
@@ -2,183 +2,236 @@ version: "3.8"
 services:
   stride1:
     image: stridezone:stride
+    user: root
```

Comment:
What's the reason for running as root?

Reply:
when volume is mounted, docker cannot access mounted files due to access rights